CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ if (NOT PLATFORM STREQUAL "ZEPHYR")
 endif()
 
 # Add compile definitions for platform and network channel specifics.
-target_compile_definitions(reactor-uc PRIVATE "PLATFORM_${PLATFORM}")
+target_compile_definitions(reactor-uc PUBLIC "PLATFORM_${PLATFORM}")
```

Comment:
The PLATFORM compile def must be visible in targets that link to reactor-uc also, because they will include the network_channel.h where it is used